Sources/WhoopDIKit/Module/AccumulationDefinition.swift

```diff
@@ -0,0 +1,121 @@
+import Foundation
+
+private func wrapAccumulationProvider<Key: AccumulationKey>(name: String?, accumulationKey: Key.Type, valueProvider: @escaping (Any?) throws -> Key.AccumulatedValue) -> (Any?, Container) throws -> Any {
```

Comment:
Nit: Let's move the private helper functions down to the bottom. Typically the most visible, exported objects should be at the top of the file, then internal stuff below that.